docs/orchestrating-elastic-stack-applications/elasticsearch/snapshots.asciidoc

```diff
@@ -285,6 +284,116 @@ PUT /_snapshot/my_s3_repository
 }
 ----
 
+[id="{p}-azure-workload-identity"]
+=== Use Azure Workload Identity
+
+Starting with version 8.16 Elasticsearch supports Azure Workload identity which allows the use of Azure blob storage for Elasticsearch snapshots without exposing Azure credentials directly to Elasticsearch. 
+
+Follow the https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster[Azure documentation] for setting up workload identity for the first five steps:
+
+. https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-a-resource-group[Create a resource group], if it does not exist yet.
+. https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-an-aks-cluster[Create] or https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#update-an-existing-aks-cluster[update] your AKS cluster to enable workload identity.
+. https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#retrieve-the-oidc-issuer-url[Retrieve the OIDC issuer URL].
+. https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-a-managed-identity[Create a managed identity] and https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-a-kubernetes-service-account[link it to a Kubernetes service account].
+. https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-the-federated-identity-credential[Create the federated identity credential].
++
+NOTE: The following steps diverge from the tutorial in the Azure documentation. However, variables initialised as part of the Azure tutorial are still assumed to be present.
++
+. Create an Azure storage account, if it does not exist yet.
++
+[source,sh,subs="attributes,callouts"]
+----
+az storage account create \
+      --name es-storage \
```

Comment:
nit: `es-storage` is not a valid storage account name:

```
Message: es-storage is not a valid storage account name. Storage account name must be between 3 and 24 characters in length and use numbers and lower-case letters only.
```

Reply:
I tried to make the names more intuitive compared to what I used for testing and failed 🤦